com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -643,11 +643,7 @@ public void NetworkUpdate(NetworkUpdateStage updateStage)
 
         private void OnNetworkEarlyUpdate()
```

Comment:
Deja Vu! Wasn't this another PR, no?

Reply:
Yes i was in a pr merged already to develop. But since its a ship blocker we either need this pr in.